### DISCUSSION TITLE
Accessing the underlying ExternalProvider

### DISCUSSION THREAD

**Discussion by theodormarcu** [wevm/wagmi#1311] [Q&A]
I know [the difference between useProvider and useSigner().provider](https://github.com/wagmi-dev/wagmi/discussions/1300). 

I was wondering if there's a way I can access the underlying `ExternalProvider` (for example, the window.ethereum) that wagmi uses.

**Comment by tmm** [comment]
Check out [`connector.getProvider()`](https://github.com/wagmi-dev/wagmi/blob/main/packages/core/src/connectors/base.ts#L55)

**Reply by johnpaulcas** [reply]
hi @tmm is that mean, we can used now the metamask as default provider? and what version is this? thanks!
